app/model/DigitalEdition.scala

```diff
@@ -0,0 +1,25 @@
+package model
+
+import configuration.Config
+
+case class DigitalEdition(id: String, name: String, price: String, cmp: String)
```

Comment:
Yeah, I believe it's really 'campaign code' - so an identifier for some given advertising campaign.